RPIPs/RPIP-33.md

```diff
@@ -147,7 +147,6 @@ possible:
 
 1. **Invite Member:** Invites a new member to the Security Council.
 2. **Remove Member:** Removes an existing member from the Security Council.
-3. **Disband:** Removes all members from the Security Council. Effectively disabling it.
```

Comment:
If we're removing this, the above should allow for removing multiple members at once (else we have a period where the security council is just one member as we remove folks one at a time). I would also suggest saying something like "Note that the security council MAY be effectively disabled by removing all members."

Reply:
No problem, Kane is going to adjust the remove function to be able to remove multiple members.